fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/models/ContainerImpl.java

```diff
@@ -48,4 +61,19 @@ public ContainerImpl(final String id, final Transaction tx, final PersistentStor
     public FedoraResource getDescribedResource() {
         return this;
     }
+
+    @Override
+    public RdfStream getTriples() {
+        try {
```

Comment:
Instead, could you use the extended class's `getTriples()` method here by changing this method to something like:
```
final Stream<Triple> extraTriples = Stream.of(
                new Triple(createURI(id), RDF.Init.type().asNode(), RDF_SOURCE.asNode()));
return new DefaultRdfStream(createURI(id), Stream.concat(super.getTriples(), extraTriples));
```

Reply:
No because the method returns a `RdfStream` and not `Stream<Triple>`. I had thought of that. I could get the `super.getTriples()` and then extract the `Stream<Triple>`. Not sure if that is easier though.

Reply:
Oh maybe I can, I had some trouble with returning Streams of triples but perhaps I was mistaken about the cause. Doing a test build.